velox/exec/tests/AggregationTest.cpp

```diff
@@ -613,5 +678,88 @@ TEST_F(AggregationTest, partialAggregationMemoryLimit) {
   assertQuery(params, "SELECT c0, count(1) FROM tmp GROUP BY 1");
 }
 
+TEST_F(AggregationTest, spill) {
```

Comment:
A single test for all of spilling logic feels not enough. Do you think it covers all the code paths?